### ISSUE TITLE
Territories that do not count for distance are shown in grey which is ambiguous when using colourblind mode

### ISSUE THREAD

**Issue by alistaircarscadden** [the-abe-train/globle#118]
Here is an example of this. Both images show France and French Guiana, which both appear when I guess France. The problem for me is that when using colourblind mode the territory shade is often extremely similar to the country shade, making it hard to judge whether it counts or not.

![1](https://user-images.githubusercontent.com/7132621/173950808-08fc6879-d3b3-4e87-82d8-5dc6de32aea5.png)

![2](https://user-images.githubusercontent.com/7132621/173950845-e0d1f442-28c2-459d-b483-c740ef7979f9.png)

**Comment by the-abe-train** [comment]
This is true. I noticed it would be an issue when I created colour blind mode, but I don't have a fix for it. I'm open to suggestions and PRs!

**Comment by noambernstein** [comment]
What about a more distinctive color (farther from the close to far gradient palette), perhaps black? Other things like blue would also be distinctive and unambiguous, but somehow I feel that “no color” makes it clear those regions don’t count.
